Client/Frontend/Browser/Tab.swift

```diff
@@ -830,6 +832,34 @@ class Tab: NSObject {
     }
 }
 
+extension Tab: UIGestureRecognizerDelegate {
+    // This prevents the recognition of one gesture recognizer from blocking another
+    func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldRecognizeSimultaneouslyWith otherGestureRecognizer: UIGestureRecognizer) -> Bool {
+        return true
+    }
+    
+    func configureEdgeSwipeGestureRecognizers() {
+        guard let webView = webView else {
+            os_log(.debug, "Tab's edge swipe gesture recognizer was never added. This will affect Tab navigation telemetry!")
```

Comment:
`os_log` is considered a [legacy logging symbols](https://developer.apple.com/documentation/os/logging/legacy_logging_symbols). When we start using iOS 14 and up we'll be able to use the new os logging API. I would suggest adding a wrapper on top of the unified logging system so we only have to change it in one place when we'll be able to.

Reply:
Great catch! Hmm given that we're iOS13, and we already have a robust logging solution (that would be good to reimplement with iOS14's changes to cut our dependency on that third party lib), maybe we can use the current log solution for now?